docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+.. _landice_ismip6_forcing:
+
+ismip6_forcing
+============
+
+The ``landice/ismip6_forcing`` test group processes (i.e., remaps and renames)
+the atmospheric and ocean forcing data of the Ice Sheet Model Intercomparison for CMIP6
+(ISMIP6) protocol. The processed data is used to force MALI in future ice sheet
+simulations. The test group includes three test cases, ``atmosphere``,``ocean_basal`` and
+``ocean_thermal``. Each test case contains one main step, called
+``ProcessSMB``, ``ProcessBasalMelt`` and ``ProcessThermalForcing``.
+Each step has the local methods (functions) of remapping and
+renaming the original ismip6 data to the format that MALI can incorporate in
+its forward simulations. In remapping the data, all test cases import the
+method ``build_mapping_file`` to create or use scrip files
+of the source (ISMIP6) and destination (MALI) mesh depending on the existence
+of a mapping file. Approximated time for processing a single forcing file
+on Cori (single core) is 2, 7, and 1 minutes for the atmosphere, ocean basal,
+and ocean thermal testcase, respectively.
+
+The ISMIP6 source data can be obtained by directly
+contacting the authors. Reference: https://www.climate-cryosphere.org/wiki/index.php?title=ISMIP6-Projections2300-Antarctica#A2.2_Retrieving_datasets_and_uploading_your_model_output
+Once the users get access to the data on Globus (https:www.globus.org), they
+need to create a base directory that is equivalent to the GHub globus endpoints
+(e.g., ``GHub-ISMIP6-Forcing`` for the 2100CE projection protocol and
+``ISMIP6-Projections-Forcing-2300`` for the 2300CE projection protocol.)
+Users also need to manually download the files in the same directory name and
+structure as on the endpoint (including the ``imbie2_basin_numbers_8km.nc``
+file in the ``Ocean_Forcing/imbie2/`` directory, which is provided in the
+globus ``GHub-ISMIP6-Forcing`` endpoint.
```

Comment:
I think this part needs quite a bit more explanation, including an example list of files for one model, scenario and end year.  For example, it isn't clear that you need only the `Regridded_8km` folder under the atmospheric forcing.  It isn't clear that you only need the atmospheric anomaly files (not the climatology) and that you don't need the files form 1950 to 1994 (I believe that's correct).

On the ocean side, it isn't clear that you only need the `1995-<end_year>` folder and not the `1850-1994`.  It's not clear if you need `temperature` and `salinity` or just `thermal_forcing`.

The `parameterizations` folder isn't mentioned explicitly either.  Since this data is only available from `GHub-ISMIP6-Forcing`, similarly to the IMBIE basins, it should probably be mentioned as well in that same sentence.  Which files are needed from `parameterizations`?

The `GHub-ISMIP6-Forcing` has `Atmosphere_Forcing` and  `Ocean_Forcing` folder with a capital `F` but `ISMIP6-Projections-Forcing-2300` uses a lowercase `f` for both.  It seems important to mention that and to state whether users need to have to match that exactly.  For the parameterization and IMBIE basins for 2300, does that mean you need an `Ocean_forcing` folder for the thermal forcing and an `Ocean_Forcing` folder for the parameterizations and IMBIE basins?